HLTrigger/Configuration/python/HLT_75e33/modules/hltHgcalLayerClustersFromSoAProducer_cfi.py

```diff
@@ -0,0 +1,13 @@
+import FWCore.ParameterSet.Config as cms
+from ..psets.hgcal_reco_constants_cfi import HGCAL_reco_constants as HGCAL_reco_constants
+
+hltHgCalLayerClustersFromSoAProducer = cms.EDProducer("HGCalLayerClustersFromSoAProducer",
+    detector = cms.string('EE'),
+    hgcalRecHitsLayerClustersSoA = cms.InputTag("hltHgcalSoARecHitsLayerClustersProducer"),
+    hgcalRecHitsSoA = cms.InputTag("hltHgcalSoARecHitsProducer"),
+    nHitsTime = cms.uint32(3),
+    src = cms.InputTag("hltHgcalSoALayerClustersProducer"),
+    timeClname = cms.string('timeLayerCluster')
+)
```

Comment:
Why the above import of HGCAL_reco_constants which are then not used in this file?